package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "async": "^2.1.2",
     "ethashjs": "~0.0.7",
-    "ethereumjs-block": "~1.6.0",
+    "ethereumjs-block": "git+https://github.com/ethereumjs/ethereumjs-block.git#byzantium-changes",
```

Comment:
Once we release ethereumjs-block Byzantium changes, we will change this line to point to the master branch of ethereumjs-block.  Later, after we successfully release, then we can release a new version of ethereumjs-block and change this line again to restrict the version.